packages/admin/blocks-admin/src/blocks/helpers/createBlockSkeleton.tsx

```diff
@@ -42,6 +42,7 @@ function common<InputApi, State, OutputApi>(): Omit<
         },
         isValid: () => true,
         previewContent: () => [],
+        resolveDependencyRoute: () => "",
```

Comment:
Is it a good idea to provide a default implementation? Might we not miss some routes?

Reply:
I think most blocks don't need their own dependencyRoutes. If the block is a ListBlock, BlocksBlock, ColumnsBlock or CompositeBlock, the routes are handled correctly. Most other blocks usually don't change the URL, I suppose (?)